.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It doesn't seem that this file association should be necessary. In any case, please update `.gitignore` to remove lines 4-7 so that this isn't part of your PR.